Cake.Recipe/Content/analyzing.cake

```diff
@@ -15,6 +14,18 @@ BuildParameters.Tasks.DupFinderTask = Task("DupFinder")
             ThrowExceptionOnFindingDuplicates = ToolSettings.DupFinderThrowExceptionOnFindingDuplicates ?? true
         };
 
+        // Workaround until 1.0.0+ version of cake is released
+        if (BuildParameters.IsDotNetCoreBuild && BuildParameters.BuildAgentOperatingSystem == PlatformFamily.Windows)
+        {
+            settings.ToolPath = Context.Tools.Resolve("jb.exe");
+            settings.ArgumentCustomization = args => args.Prepend("dupfinder");
+        }
+        else if(BuildParameters.IsDotNetCoreBuild)
+        {
+            settings.ToolPath = Context.Tools.Resolve("jb");
+            settings.ArgumentCustomization = args => args.Prepend("dupfinder");
+        }
+
```

Comment:
We'd probably want an else if statement here for when the user isn't running on windows, to warn the user and not continuing with the task.

Or we could update the `WithCriteria` function you removed: ie
```
WithCriteria(() => BuildParameters.IsDotNetCoreBuild || BuildParameters.BuildAgentOperatingSystem == PlatformFamily.Windows
```

Reply:
👍